lib/database/database.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:firebase_database/firebase_database.dart';
+
+//https://www.youtube.com/watch?v=sXBJZD0fBa4&ab_channel=Firebase
+
+class WriteDatabase extends StatefulWidget {
```

Comment:
I'm kind of confused on how this class will work. So we also will need a class for reading from the db right?

Reply:
Yes, reading and writing will be different classes, I also need to either initialize all beastie data via the app or just write all of it in and then users can retrieve as needed.